ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPoolReplacementHandler.java

```diff
@@ -28,7 +28,10 @@
   private final List<TransactionPoolReplacementRule> rules;
 
   public TransactionPoolReplacementHandler(final Percentage priceBump) {
-    this(asList(new TransactionReplacementByPriceRule(priceBump)));
+    this(
+        asList(
```

Comment:
I wonder if the rules list can be defined based on the fork as we do for the rest here
https://github.com/hyperledger/besu/blob/1772f8b1f89686e674e3ccaf333ae7f6355d1d36/ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecs.java#L458

Reply:
I pushed a commit that plumbs protocolSchedule down to TransactionPoolReplacementHandler.  LMK what you think, I am on the fence about it.